database/tables/plan_grants.pg

```diff
@@ -10,10 +9,9 @@ columns
 
 indexes
     plan_grants_pkey: PRIMARY KEY (id) USING btree (id)
-    plan_grants_by_name_idx: UNIQUE NULLS NOT DISTINCT (plan_name, institution_id, course_instance_id, enrollment_id) USING btree (plan_name, institution_id, course_instance_id, enrollment_id) NULLS NOT DISTINCT
+    plan_grants_institution_course_instance_user_name_idx: UNIQUE NULLS NOT DISTINCT (institution_id, course_instance_id, user_id, plan_name) USING btree (institution_id, course_instance_id, user_id, plan_name) NULLS NOT DISTINCT
```

Comment:
Note how this index changed: I moved `plan_name` from the first component of the index to the last, as our query pattern is generally going to be "give me all the plan grants associated with XYZ, regardless of which plan it it". I think this new index is what we want?

Reply:
Yep, that makes sense.